### PR TITLE
Add support to build Creator Kit 2 application.

### PR DESCRIPTION
**Pull request by manohar-narkhede-imgtec**
Signed-off-by: Manohar Narkhede <Manohar.Narkhede@imgtec.com>

### CODE REVIEW

File: creator-kit-2-cascoda.config [hunk 1]
```diff
@@ -0,0 +1,15 @@
+CONFIG_TARGET_pistachio=y
+CONFIG_TARGET_pistachio_marduk=y
+CONFIG_TARGET_pistachio_marduk_marduk_ca8210=y
+CONFIG_PACKAGE_motion-led-controller=y
+CONFIG_PACKAGE_webscripts=y
+CONFIG_IMAGEOPT=y
+CONFIG_VERSIONOPT=y
+CONFIG_VERSION_DIST="OpenWrt"
+CONFIG_VERSION_NICK="Ci40Kit1Project1"
```
Comment (nikhil-zinjurde-imgtec) [inline creator-kit-2-cascoda.config]: Ci40Kit1Project2

File: creator-kit-2.config [hunk 2]
```diff
@@ -0,0 +1,15 @@
+CONFIG_TARGET_pistachio=y
+CONFIG_TARGET_pistachio_marduk=y
+CONFIG_TARGET_pistachio_marduk_marduk_cc2520=y
+CONFIG_PACKAGE_motion-led-controller=y
+CONFIG_PACKAGE_webscripts=y
+CONFIG_IMAGEOPT=y
+CONFIG_VERSIONOPT=y
+CONFIG_VERSION_DIST="OpenWrt"
+CONFIG_VERSION_NICK="Ci40Kit1Project1"
```
Comment (nikhil-zinjurde-imgtec) [inline creator-kit-2.config]: Ci40Kit1Project2

File: creator-kit-2-cascoda.config [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+CONFIG_VERSION_NUMBER="dev"
+CONFIG_VERSION_REPO="https://downloads.creatordev.io/pistachio/marduk"
```
Comment (nikhil-zinjurde-imgtec) [inline creator-kit-2-cascoda.config]: https://downloads.creatordev.io/pistachio/marduk/packages

File: creator-kit-2.config [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+CONFIG_VERSION_NUMBER="dev"
+CONFIG_VERSION_REPO="https://downloads.creatordev.io/pistachio/marduk"
```
Comment (nikhil-zinjurde-imgtec) [inline creator-kit-2.config]: https://downloads.creatordev.io/pistachio/marduk/packages

File: Makefile [hunk 5]
```diff
@@ -49,20 +49,26 @@ else
...
+	@if [ $(KIT_APP_NO) -eq "1" ]; then $(MAKE) -C $(DIR__CKT)/packages/button-sensor TARGET=mikro-e USE_CA8210=1; fi
+	@if [ $(KIT_APP_NO) -eq "2" ]; then $(MAKE) -C $(DIR__CKT)/packages/motion-sensor TARGET=mikro-e USE_CA8210=1; fi
 else
-	$(MAKE) -C $(DIR__CKT)/packages/button-sensor TARGET=mikro-e USE_CC2520=1
+	@if [ $(KIT_APP_NO) -eq "1" ]; then $(MAKE) -C $(DIR__CKT)/packages/button-sensor TARGET=mikro-e USE_CC2520=1; fi
+	@if [ $(KIT_APP_NO) -eq "2" ]; then $(MAKE) -C $(DIR__CKT)/packages/motion-sensor TARGET=mikro-e USE_CC2520=1; fi
 endif
 
 # Copy files to build/output/
 copy_contiki:
```
Comment (abhijit-mahajani-imgtec) [inline Makefile]: get_kit_app dependency is needed

File: Makefile [hunk 6]
```diff
@@ -49,20 +49,26 @@ else
 	$(MAKE) $(SUBMAKEFLAGS) -C $(DIR__OPENWRT) -j$(J)
 endif
 
+get_kit_app:
+	@$(eval KIT_APP_NO=$(shell echo $P | tr -cd [:digit:]))
+
 # Building Contiki apps
 .PHONY: build_contiki
 build_contiki:
```
Comment (abhijit-mahajani-imgtec) [inline Makefile]: get_kit_app dependency is needed

**Comment by abhijit-mahajani-imgtec** [comment]
Approved

**Comment by abhijit-mahajani-imgtec** [comment]
Verified

**Pull request merged**